setup.py

```diff
@@ -1,35 +0,0 @@
-from setuptools import find_packages
```

Comment:
since this ros package wasn't using `ament_python`, I don't think most of this file is ever being properly parsed and used.  at the very least `data_files` and `entry_points` isn't being handled and all that functionality here is already taken care of in the CMakelists so I thought I should remove this misleading file. All of this info is already duplicated elsewhere.